### PR TITLE
Nazar/feature/move without material UI

### PR DESCRIPTION
**Pull request by Nazar-kob**
We currently have three pages. However, data validation is not yet implemented for the last page
image
image
image
image

### CODE REVIEW

File: frontend/src/App.css [hunk 1]
```diff
@@ -0,0 +1,171 @@
+html,
```
Comment (SvyatoslavArtymovych) [inline frontend/src/App.css:1]: Would "*" be a better option here?
* will aplly styles for all html tags

**Review by Taraslysak — CHANGES_REQUESTED** [review]
Good job! But please consider some minor refactoring

File: frontend/src/App.css (hunk 1 above)
Comment (Taraslysak) [reply frontend/src/App.css:1]: Quite possible. Definitely worth trying

**Review by kipod — APPROVED** [review]
Thank you

File: frontend/src/App.css (hunk 1 above)
Comment (Nazar-kob) [reply frontend/src/App.css:1]: done

File: frontend/package.json [hunk 2]
```diff
@@ -11,12 +11,17 @@
   "dependencies": {
     "@emotion/react": "^11.10.5",
     "@emotion/styled": "^11.10.5",
+    "@material-ui/styles": "^4.11.5",
```
[the remark this answers is not in the document]
Comment (Nazar-kob) [reply frontend/package.json:14]: Thanks, good advice

File: frontend/src/App.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (Nazar-kob) [reply frontend/src/App.css:1]: done)

File: frontend/src/pages/Market/Market.tsx [hunk 3]
```diff
@@ -1,44 +1,75 @@
...
-
-  const [showProducts, setShowProducts] = useState<boolean>(false);
-  const handleStartOrder = () => {
-    setShowProducts(true);
+
+  const [cartState, dispatchCart] = useReducer(reducer, initialState);
+
+  const [step, setStep] = useState<BusinessStep>(BusinessStep.START_ORDER);
+
+  const [hiddenBtn, setHiddenBtn] = useState<boolean>(false);
```
[the remark this answers is not in the document]
Comment (Nazar-kob) [reply frontend/src/pages/Market/Market.tsx:35]: Thanks I got you

File: frontend/src/pages/Market/components/ProductItem/ProductType.tsx [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+export function ProductType({
+  soldBy,
+  amount,
+  setAmount,
+  selectType,
+  setSelectType,
+}: PrepProductType) {
+  const handleSelectKilogram = () => {
+    setSelectType(ItemUnit.kilogram);
+  };
```
[the remark this answers is not in the document]
Comment (Nazar-kob) [reply frontend/src/pages/Market/components/ProductItem/ProductType.tsx:23]: I got you

File: frontend/src/pages/Market/components/ProductList/ProductList.tsx [hunk 5]
```diff
@@ -1,53 +1,73 @@
...
+      return data.products;
+    },
   });
+
+  const onProductClicked = (id: number) => {
+    const selectedItem = data?.find((item) => item.id === id);
+    if (selectedItem) {
+      console.log(id);
+    }
+  };
```
[the remark this answers is not in the document]
Comment (Nazar-kob) [reply frontend/src/pages/Market/components/ProductList/ProductList.tsx:33]: Yes

**Pull request merged**